browser/components/newtab/tests/browser/dummy_page.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+
+<html>
```

Comment:
please add after `<html>`: 
```
<meta charset="utf-8">
```

Stops the annoying warning about lack of content type in the console.